hypervisor/arch/x86/configs/whl-phx-i7/board.c

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (C) 2019 Intel Corporation. All rights reserved.
```

Comment:
```suggestion
 * Copyright (C) 2020 Intel Corporation. All rights reserved.
```

Reply:
Are these `*.c/*.h` fiels auto-generated? If so, we may need to update the tool that does it to include the correct year.